open-api/typescript-sdk/src/fetch-client.ts

```diff
@@ -2852,6 +2857,10 @@ export enum AssetOrder {
     Asc = "asc",
     Desc = "desc"
 }
+export enum Role {
```

Comment:
Why is this here?

Reply:
It was added by running `make open-api`.